### DISCUSSION TITLE
General ESIPT e911 Pattern Questions

### DISCUSSION THREAD

**Discussion by ttucker-twilion** [twilio-labs/paste#1850] [Office hours] [answered]
Hi,

I'm working on some designs to support e911 emergency calling efforts in the SIP trunking product. I wanted to discuss some best practices related to button consistencies, filter patterns, and page transitions after submitting changes in the console.  I also wanted to chat about patterns that I've heard are possibly on the roadmap for future releases, including file upload and multi-step progress indicator/stepper. 

Here's a link to my work in progress: https://www.figma.com/file/UItWdasataJ3YhMLsXxn9p/2021-Q2-ESIPT-e911-Emergency-Calling?node-id=161%3A53218

**Comment by nkrantz** [comment, marked as answer]
Hi @ttucker-twilion! Just following up here with the Office Hours recording. [You can watch here](https://www.loom.com/share/49f195ad53cc40f28dfc360d9d2d8a8b), it's password protected but you can find the password in #help-design-system
